napari/plugins/__init__.py

```diff
@@ -1,34 +1,49 @@
+from functools import lru_cache
 from pathlib import Path
 
 from npe2 import PluginManager as _PluginManager
 from npe2 import PluginManifest
 from npe2.manifest.package_metadata import PackageMetadata
 
 from ..settings import get_settings
+from . import _npe2
 from ._plugin_manager import NapariPluginManager
 
 __all__ = ["plugin_manager", "menu_item_template"]
 
-_npe2pm = _PluginManager.instance()
-
-# this is a workaround for the fact that briefcase does not seem to
-# include napari's entry_points.txt in the bundled app, so the builtin plugins
-# don't get detected.  So we just register it manually.  This could potentially
-# be removed when we move to a different bundle strategy
-if 'napari' not in _npe2pm._manifests:
-    mf_file = Path(__file__).parent.parent / 'builtins.yaml'
-    mf = PluginManifest.from_file(mf_file)
-    mf.package_metadata = PackageMetadata.for_package('napari')
-    _npe2pm.register(mf)
-
+#: Template to use for namespacing a plugin item in the menu bar
+menu_item_template = '{}: {}'
 # the main plugin manager instance for the `napari` plugin namespace.
 plugin_manager = NapariPluginManager()
-plugin_manager._initialize()
 
-# Disable plugins listed as disabled in settings, or detected in npe2
-_from_npe2 = {m.package_metadata.name for m in _npe2pm._manifests.values()}
-_toblock = get_settings().plugins.disabled_plugins.union(_from_npe2)
-plugin_manager._blocked.update(_toblock)
 
-#: Template to use for namespacing a plugin item in the menu bar
-menu_item_template = '{}: {}'
+@lru_cache()  # only call once
```

Comment:
```suggestion
@cache()  # only call once no need for LRU
```

Reply:
Actually sorry, that's 3.9 and above

Reply:
Ah you are right, never mind.

Reply:
I sometime am like "I how we can use X soon" and X is since Python 3.4, and sometime it's the opposite and it's like a super new feature

Reply:
lol